automation/run-tests.sh

```diff
@@ -13,10 +13,23 @@ TEST_TYPE_LINT="lint"
 TEST_TYPE_UNIT_PY36="unit_py36"
 TEST_TYPE_UNIT_PY38="unit_py38"
 TEST_TYPE_INTEG="integ"
+TEST_TYPE_INTEG_SLOW="integ_slow"
```

Comment:
How about renaming integ to integ_fast and have an integ type that would run everything (could be added in the future).

Reply:
I can live with this.